docs/user/beyond/fulltext.rst

```diff
@@ -14,6 +14,9 @@ Introduction
 
 Full-text search is for searching a single stored document which is distinguished from regular search based on original texts in database. It tries to match search criteria by examining all of the words in each document. In OpenSearch, full-text queries provided enables you to search text fields analyzed during indexing.
 
+This document describes the original SQL engine. It only applies to queries that are not handled by the current SQL engine. Documentation for the current engine can be found  `here <../dql/functions.rst#relevance>`_.
```

Comment:
I'm just thinking we could use slightly more accurate wording here, is this better? 
`It only applies to queries that cannot be handled by the current SQL engine.`